docs/examples/robust_paper/utils.py

```diff
@@ -112,3 +113,13 @@ def get_mle_params_and_guide(conditioned_model, n_iters=2000, lr=0.03):
         k: v.clone().detach().requires_grad_(True) for k, v in guide_train().items()
     }
     return theta_hat, MLEGuide(theta_hat)
+
+
+@contextmanager
```

Comment:
Can you just use `reset_rng_state` from `chirho.robust.internals.utils` here instead?